docs/sql-standards.md

```diff
@@ -137,6 +137,51 @@ UPDATE `creature_template` SET `mechanic_immune_mask`=`mechanic_immune_mask`|64|
 UPDATE `creature_template` SET `mechanic_immune_mask`=`mechanic_immune_mask`&~(64|256|1024) WHERE `entry` = 7727;
 ```
 
-### Note for SQL reviewer
+## Tables and Columns
+
+### Integers
+
+We do not define the width of an integer when we create new columns. (Width is deprecated in later versions of MySQL 8)
+
+```
+TINYINT(M)   -> TINYINT
+SMALLINT(M)  -> SMALLINT
+INT(M)       -> INT
+MEDIUMINT(M) -> MEDIUMINT
+BIGINT(M)    -> BIGINT
+
+BOOL         -> Never, synonym for TINYINT. 0 = false <>0 = true
```

Comment:
does it mean that we should never use BOOL or what? I think it should be more clear

Reply:
done